inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/InAppMessagingSpec.kt

```diff
@@ -255,13 +261,13 @@ class InAppMessagingSpec : BaseTest() {
         When calling viewGroup.tag itReturns "1"
     }
 
-    private fun initializeInstance() {
+    private fun initializeInstance(shouldDisableCaching: Boolean = false) {
         WorkManagerTestInitHelper.initializeTestWorkManager(ApplicationProvider.getApplicationContext())
         Settings.Secure.putString(
                 ApplicationProvider.getApplicationContext<Context>().contentResolver,
                 Settings.Secure.ANDROID_ID,
                 "test_device_id")
         InAppMessaging.init(ApplicationProvider.getApplicationContext(), "test", "",
-                isDebugLogging = true, isForTesting = true)
+                isDebugLogging = true, isForTesting = !shouldDisableCaching)
```

Comment:
doesn't `isForTesting` make cache disable if set to true? Shouldn't that flag be without negation?

Reply:
This should be fine since this is negated in `InAppMessaging.init()`.
But add new variable to avoid conflict.